cli/popper/commands/cmd_run.py

```diff
@@ -130,11 +130,26 @@ def update_badge(status):
             'status': status
         }
         try:
-            r = requests.post(badge_server_url, data=data)
-            if r.status_code != 201:
-                pu.warn("Could not create a record on the badge server")
-        except requests.exceptions.RequestException:
-            pu.warn("Could not communicate with the badge server")
+            commit_id = check_output(['git', 'rev-parse', 'HEAD'])[:-1]
```

Comment:
there's a [`get_head_commit()`](https://github.com/systemslab/popper/blob/master/cli/popper/utils.py#L291) function available now, can we use that?